Node/core/src/bots/ChatConnector.ts

```diff
@@ -181,95 +191,131 @@ export class ChatConnector implements IConnector, IBotStorage {
             var verifyOptions: jwt.VerifyOptions;
             var openIdMetadata: OpenIdMetadata;
             const algorithms: string[] = ['RS256', 'RS384', 'RS512'];
+            
+            if(this.settings.enableSkills && SkillValidation.isSkillToken(authHeaderValue)) {
+                const skillMsg = utils.clone(req.body);
+                this.prepIncomingMessage(skillMsg);
+
+                let authConfiguration = this.settings.authConfiguration;
```

Comment:
Avoid using `let` when it can be presented via a ternary statement

`const authConfiguration = this.settings.authConfiguration || new DefaultAuthenticationConfiguration(this.settings.allowedCallers);`

Reply:
fixed